.eslintrc.js

```diff
@@ -22,10 +22,10 @@ module.exports = {
     'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
     'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
     'max-len': ['warn', { code: 120, ignoreRegExpLiterals: true }],
-    // 'vue/attribute-hyphenation': 'off',
-    // 'vue/no-deprecated-filter': 'warn',
+    'vue/attribute-hyphenation': 'off',
```

Comment:
I don't want the eslint plugins to auto-fix `myAttribute=` to `my-attribute=`.

Reply:
@severinbeauvais can you expand on your thinking here?  Would you prefer to use `myAttribute` going forward or switch to `my-attribute` but not immediately?

Reply:
We can change this later but it works as-is and I didn't want to change too many things right now if possible.

I also didn't like that it changed `payURL` to `pay-u-r-l` etc 😁 We can fix this later.

Reply:
Good call, let's not change too many things in a single PR, but going forward I think it's very important that we follow any guidance from Vue that's "strongly recommended"

Reply:
Agreed.

Best case: it works in Vue3 and we should fix it sooner or later.
Worst case: it doesn't work in Vue3 and I'll need to fix it asap.